libs/platform/unit/platform_unit_test.cpp

```diff
@@ -931,13 +929,24 @@ TEST_CASE("ring_buffer_reserve_submit_discard", "[platform]")
 
     uint8_t* mem1 = nullptr;
     REQUIRE(ebpf_ring_buffer_reserve(ring_buffer, &mem1, 10) == EBPF_SUCCESS);
-    _Analysis_assume_(mem1 != nullptr);
-    ebpf_ring_buffer_submit(mem1);
+    REQUIRE(mem1 != nullptr);
+    ebpf_result_t result = ebpf_ring_buffer_submit(mem1);
+    // Workaround for code analysis failure:
+    // C28193: 'result' holds a value that must be examined.
+    if (result != EBPF_SUCCESS) {
+        REQUIRE(result == EBPF_SUCCESS);
```

Comment:
Can we simply have `REQUIRE(result == EBPF_SUCCESS);` instead of `if (result != EBPF_SUCCESS)` check?

Reply:
For reasons that are still TBD REQUIRE macro isn't recognized as inspecting the result value.